addons/sys_gsa/CfgVehicles.hpp

```diff
@@ -0,0 +1,183 @@
+                };
+
+                class ACRE_connect {
+                    selection = "";
+                    displayName = CSTRING(connect);
+                    distance = 10;
+                    condition = QUOTE(!([ARR_2(_player,_target)] call DFUNC(isAntennaConnected)));
+                    statement = QUOTE(true);
+                    insertChildren = QUOTE([ARR_2(_player,_target)] call DFUNC(connectChildrenActions));
+                    showDisabled = 0;
+                    exceptions[] = {};
+                    priority = 5;
+                    icon = "\idi\acre\addons\ace_interact\data\icons\antenna.paa";
+                };
+
+                class ACRE_disconnect {
+                    selection = "";
+                    displayName = CSTRING(disconnect);
+                    distance = 10;
+                    condition = QUOTE([ARR_2(_player,_target)] call DFUNC(isAntennaConnected));
+                    statement = QUOTE([ARR_2(_player,_target)] call DFUNC(disconnect));
+                    showDisabled = 0;
+                    exceptions[] = {};
+                    priority = 5;
+                    icon = "\idi\acre\addons\ace_interact\data\icons\antenna.paa";
+                };
+            };
+        };
+    };
+
+    class vhf30108spike: House {
+        class EventHandlers {
+            class CBA_Extended_EventHandlers: CBA_Extended_EventHandlers {};
+        };
+
+        scope = 2;
+        scopeCurator = 2;
+        scopeArsenal = 2;
+        displayName = "ACRE VHF30108 GS";
+        model = QPATHTOF(data\vhf30108\models\vhf30108spike.p3d);
+        icon = QPATHTOF(data\vhf30108\ui\icon_antenna_ca.paa);
+        editorPreview = QPATHTOF(data\vhf30108\ui\spike.jpg);
+        vehicleClass = "Items";
+
+        class AcreComponents {
+            componentName = "ACRE_243CM_VHF_TNC";
+            mountedAntenna = "";
+            mastMount[] = {"vhf30108Item"};
+        };
+
+        class ACE_Actions {
+            class ACE_MainActions {
+                selection = "interaction_point";
+                distance = 10;
+                condition = "(true)";
+
+                class ACRE_pickup {
+                    selection = "";
+                    displayName = CSTRING(pickUp);
+                    distance = 10;
+                    condition = "true";
+                    statement = QUOTE([ARR_2(_player,_target)] call DFUNC(pickUp));
+                    showDisabled = 0;
+                    exceptions[] = {};
+                    priority = 5;
+                    //icon = "\idi\acre\addons\ace_interact\data\icons\antenna.paa";
+                };
+
+                class ACRE_mountMast {
+                    selection = "";
+                    displayName = CSTRING(mountMast);
+                    distance = 10;
+                    condition = QUOTE([ARR_2(_player), "ACRE_VHF30108MAST"] call DEFUNC(sys_core,hasItem));
```

Comment:
Usage of ARR doesn't look correct (`)` in wrong place) and double quotes inside `QUOTE`.

I suggest changing to:
```suggestion
                    condition = '[_player, "ACRE_VHF30108MAST"] call DEFUNC(sys_core,hasItem)';
```